test/integration/queries/pages/dehydrated-state.tsx

```diff
@@ -27,11 +26,7 @@ function Content() {
 }
 
 function DehydratedState() {
-  return (
-    <Suspense fallback="Loading ...">
-      <Content />
-    </Suspense>
-  )
+  return <Content />
```

Comment:
I'm pretty sure suspense boundary is needed here for client navigation

Reply:
actually maybe not..

Reply:
I just checked to be sure, and client navigation works fine without suspense